tensorflow/python/ops/image_ops_impl.py

```diff
@@ -5536,6 +5536,20 @@ def generate_bounding_box_proposals(scores,
                                     name=None):
   """Generate bounding box proposals from encoded bounding boxes.
 
+  Args:
+    scores: A 4-D float tensor of shape `[num_images, height, width, num_achors]` 
+      containing scores of the boxes for given anchors, can be unsorted.
+    bbox_deltas: A 4-D float tensor of shape `[num_images, height, width, 4 x num_anchors]`. 
+      encoding boxes with respec to each anchor. Coordinates are given in the form [dy, dx, dh, dw].
```

Comment:
Encoding should be capital. [dy, dx, dh, dw] --> `[dy, dx, dh, dw]`

Reply:
@ashwin-phadke Not sure if 'encoding boxes with respec' should be a new sentence since there's a period before that line. If it is can you capitalize encoding.

Reply:
It isn't a new sentence. Will change that.